test/Microsoft.IdentityModel.TestUtils/TestSets.cs

```diff
@@ -990,7 +992,7 @@ public static SignedInfoTestSet UnknownReferenceTransform
                                 unknownTransform,
                                 SecurityAlgorithms.ExclusiveC14n,
                                 SecurityAlgorithms.Sha256Digest,
-                                Default.ReferenceDigestValue))
+                                Default.ReferenceDigestValue, typeAttr))
```

Comment:
super nit: same as above
```suggestion
                                Default.ReferenceDigestValue, 
                                typeAttr))
```

Reply:
Thank you so much. I will resolve it